<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Go_Online</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/Go.ico</normaloff>
    <normalon>../Images/Go.ico</normalon>
    <disabledoff>../Images/Go.ico</disabledoff>
    <disabledon>../Images/Go.ico</disabledon>
    <activeoff>../Images/Go.ico</activeoff>
    <activeon>../Images/Go.ico</activeon>
    <selectedoff>../Images/Go.ico</selectedoff>
    <selectedon>../Images/Go.ico</selectedon>../Images/Go.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label_ID">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>Mail ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Password">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>PIN ：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>402433422@qq.com</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string> Please enter your Mail.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Passworld">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>580231</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string> Please enter your PIN.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbutton_Register">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>260</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Retrieve">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>RePIN</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Login">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Quit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
